startup/99-gui-ht.py

```diff
@@ -716,8 +717,15 @@ def main_plan(file_name):
             if file_name is None:
                 gui_path = self.path_select.path
                 if gui_path and reason:
+                    if '/' in reason:
+                        raise Exception('File name cannot include reserved character "/".')
```

Comment:
Maybe there can be a more scoped custom exception here?

Such as:
```py
class FileInvalidException(Exception):
    pass
```

The same can be useful for the other 2 exceptions below.

Reply:
Makes sense, done.